app/views/angular_template/test.html

```diff
@@ -0,0 +1,2 @@
+<!-- Please do not remove, will break tests :) -->
+It works!
```

Comment:
Can we just use one of the real templates in the test? That's also not perfect, but seems less bad.